openshiftcontrolplane/v1/types.go

```diff
@@ -177,6 +177,8 @@ type BuildControllerConfig struct {
 
 	BuildDefaults  *BuildDefaultsConfig  `json:"buildDefaults"`
 	BuildOverrides *BuildOverridesConfig `json:"buildOverrides"`
+
+	TrustedCABundle string `json:"trustedCABundle"`
```

Comment:
Ultimate destination for PEM-encoded CA bundle that is mounted into build pods.

Reply:
why `additionalTrustedCA` above and `trustedCABundle` here?  Is it no longer the additional one?  Is it the only?  Is there any way to exclude the system roots?

Also, add godoc for this struct while you're here.  For this field, indicate what empty means.

Reply:
+1 on naming consistency.

I think we can use AdditionalTrustedCA everywhere.  The CAs will be used in addition to the system certs that exist in the builder image, you cannot remove/replace the system certs.

Reply:
I have worry about the 3.11 -> 4.0 upgrade path when it comes to the image policy config. For 3.11 this is a reference to a file on disk. For 4.0 the string will contain the actual PEM-encoded content.

Reply:
My expectation is the migration should take the 3.11 on disk content, put it into a configmap key and point Build.Spec.AdditionalTrustedCA to that configmap key.  (ie as if the admin had set it up from scratch on 4.0).

Reply:
> For 4.0 the string will contain the actual PEM-encoded content.

Definitely write godoc.  This should reference to a file on disk.

Reply:
> This should reference to a file on disk.

why are we not inlining the content into the config.yaml?

Reply:
this makes sense at a high level. There a few details I think will warrant discussion in later PRs.